### PR TITLE
Add styles for rows in user table

### PR DESCRIPTION
**Pull request by noahtalerman**
- Add status indicators for the new "Users" table
- Make "Actions" dropdown display above the table

### CODE REVIEW

File: frontend/components/TableContainer/DataTable/_styles.scss [hunk 1]
```diff
@@ -57,29 +57,34 @@
...
+      &:before {
+        border-radius: 100%;
+        content: ' ';
+        display: inline-block;
+        height: 8px;
+        margin-right: 8px;
+        width: 8px;
+      }
+
+      &--online, &--active {
```
Comment (ghernandez345) [inline frontend/components/TableContainer/DataTable/_styles.scss:69]: hey lets change this implementation and move these styles out of this file. DataTable is a generic table that shows the data it is passed. It has no concept of what `offline` `online` and `invite pending` are. Let's try to move these into the `_styles.scss` files of  `ManageHostPage` and `UserManagementPage` respectively, as that is where those statuses have more of a meaning.

File: frontend/components/TableContainer/DataTable/StatusCell/StatusCell.tsx [hunk 2]
```diff
@@ -7,9 +7,16 @@ interface IStatusCellProps {
 
 const StatusCell = (props: IStatusCellProps): JSX.Element => {
   const { value } = props;
+
+  const generateClassTag = (rawValue: string) => {
```
Comment (ghernandez345) [inline frontend/components/TableContainer/DataTable/StatusCell/StatusCell.tsx:11]: Lets be explicit on the return value type. Out linting will throw a warning without it. so this should be:

```javascript
const generateClassTag = (rawValue: string): string => {}
```

File: frontend/components/forms/fields/Dropdown/_styles.scss [hunk 3]
```diff
@@ -149,8 +149,8 @@
   .Select-option {
     color: $core-dark-blue-grey;
     font-size: $x-small;
-    margin: 6px 0;
-    padding: 6px 16px;
+    margin: 8px;
+    padding: 10px;
```
Comment (ghernandez345) [inline frontend/components/forms/fields/Dropdown/_styles.scss:153]: we can improve this if we use the scss variables if possible

File: frontend/components/TableContainer/DataTable/DropdownCell/_styles.scss [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+      .Select-input {
+        height: auto;
+      }
+    }
+
+    .Select-menu-outer {
+    margin-top: 4px;
+    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
+    border-radius: 4px;
```
Comment (ghernandez345) [inline frontend/components/TableContainer/DataTable/DropdownCell/_styles.scss:25]: there is a scss variable for border radiuses (is it radii???) i think.

**Review by ghernandez345 — CHANGES_REQUESTED** [review]

File: frontend/components/TableContainer/DataTable/StatusCell/StatusCell.tsx (hunk 2 above)
Comment (noahtalerman) [reply frontend/components/TableContainer/DataTable/StatusCell/StatusCell.tsx:11]: Got it. Ty for the typescript guidance.

**Comment by noahtalerman** [comment]
@ghernandez345 my latest commits refactor the data table styles by eliminating duplicate and unused styles from `UserManagementPage` and `ManageHostsPage`
- Move status styles to respective stylesheets for `UserManagementPage` and `ManageHostsPage`
- Add return type for `GenerateClassTag()`
- Add scss variables in `Dropdown`. I also made minor updates to these values. More changes to variables are to come with the Aligning Figma components with Fleet UI project (fleetdm/fleet/issues/539)
- Add border-radius variable

File: frontend/pages/admin/UserManagementPage/_styles.scss [hunk 5]
```diff
@@ -8,75 +8,6 @@
     padding-bottom: $pad-medium;
   }
 
-  &__wrapper {
```
Comment (ghernandez345) [inline frontend/pages/admin/UserManagementPage/_styles.scss:11]: nice thanks for cleaning up

**Review by ghernandez345 — APPROVED** [review]

**Pull request merged**